ext/src/ruby_api/store.rs

```diff
@@ -163,40 +163,29 @@ impl Store {
     }
 
     /// @yard
-    /// Returns the amount of fuel consumed by this {Store}’s execution so far,
-    /// or +nil+ when the {Engine}’s config does not have fuel enabled.
-    /// @return [Integer, Nil]
-    pub fn fuel_consumed(&self) -> Option<u64> {
-        self.inner_ref().fuel_consumed()
+    /// Returns the amount of fuel in the {Store}.
+    ///
+    /// @return [Integer]
+    /// @raise [Error] if fuel consumption is not enabled via
+    /// Wasmtime::Engine::new.
```

Comment:
Looks like Yard does not pick up `Wasmtime::Engine::new` like this, maybe it needs to be indented or on the same line? <img width="543" alt="Screenshot 2024-01-05 at 1 07 33 PM" src="https://github.com/bytecodealliance/wasmtime-rb/assets/224311/04b08d74-959b-4c05-99a9-c36b439969a6">

This seems to do it for me:

```suggestion
    /// @raise [Error] if fuel consumption is not enabled via
    ///   {Wasmtime::Engine.new}.
```

- using `{}` will link to the symbol
- while `::` is valid for singleton method call, Yard does not undersand it and I think `.` is more common in the community

Same below.